src/main/java/it/gov/pagopa/payhub/activities/dto/exportflow/debtposition/IUVInstallmentsExportFlowFileDTO.java

```diff
@@ -0,0 +1,93 @@
+package it.gov.pagopa.payhub.activities.dto.exportflow.debtposition;
+
+import com.opencsv.bean.CsvBindByName;
+import com.opencsv.bean.CsvDate;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class IUVInstallmentsExportFlowFileDTO {
+
+    @CsvBindByName(column = "IUD")
+    private String iud;
+
+    @CsvBindByName(column = "codIUV")
+    private String iuv;
+
+    @CsvBindByName(column = "tipoIdentificativoUnivoco")
+    private EntityTypeEnum entityType;
+
+    @CsvBindByName(column = "codiceIdentificativoUnivoco")
+    private String fiscalCode;
+
+    @CsvBindByName(column = "anagraficaPagatore")
+    private String fullName;
+
+    @CsvBindByName(column = "indirizzoPagatore")
+    private String address;
+
+    @CsvBindByName(column = "civicoPagatore")
+    private String civic;
+
+    @CsvBindByName(column = "capPagatore")
+    private String postalCode;
+
+    @CsvBindByName(column = "localitaPagatore")
+    private String location;
+
+    @CsvBindByName(column = "provinciaPagatore")
+    private String province;
+
+    @CsvBindByName(column = "nazionePagatore")
+    private String nation;
+
+    @CsvBindByName(column = "emailPagatore")
+    private String email;
+
+    @CsvBindByName(column = "dataEsecuzionePagamento")
+    @CsvDate(value = "yyyy-MM-dd")
+    private LocalDate dueDate;
+
+    @CsvBindByName(column = "importoDovuto")
+    private BigDecimal amount;
+
+    @CsvBindByName(column = "tipoDovuto") 
+    private String debtPositionTypeCode;
+
+    @CsvBindByName(column = "causaleVersamento")
+    private String remittanceInformation;
+
+    @CsvBindByName(column = "datiSpecificiRiscossione")
+    private String legacyPaymentMetadata;
+
+    @CsvBindByName(column = "bilancio")
+    private String balance;
+
+    @CsvBindByName(column = "flgGeneraIuv")
+    private Boolean flagPuPagoPaPayment;
+
+    @CsvBindByName(column = "tipoVersamento")
+    private String paymentType;
+
+    @CsvBindByName(column = "commissioneCaricoPa")
+    private BigDecimal paCommissionCents;
+
+    @CsvBindByName(column = "azione")
+    private ActionEnum action;
+
+    public enum ActionEnum {
+        I, M, A
+    }
+
+    public enum EntityTypeEnum {
+        F, G
+    }
```

Comment:
let's re-use EntityTypeEnum instead of defining a new one.
on the mapper you could next simple use it